joern-cli/frontends/javasrc2cpg/src/test/scala/io/joern/javasrc2cpg/querying/TypeTests.scala

```diff
@@ -155,6 +155,9 @@ class TypeTests extends JavaSrcCodeToCpgFixture {
   }
 
   "should default to ANY with a matching type node for unresolved types" in {
+    val xxx = cpg.typ.fullName("ANY").l
+    val yyy = cpg.typ.filter(_ => true).fullName("ANY").l
+    val zzz = 1 + 1
```

Comment:
what's this?

Reply:
It's debug nonsense, thanks for spotting, removed.
(intellij insists on some lines with instructions in order to set breakpoints)